data/data-pipeline/data_pipeline/tests/sources/cdc_life_expectancy/test_etl.py

```diff
@@ -0,0 +1,112 @@
+# pylint: disable=protected-access
+import pathlib
+from unittest import mock
+import requests
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.etl.sources.cdc_life_expectancy.etl import CDCLifeExpectancy
+from data_pipeline.tests.sources.example.test_etl import TestETL
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class TestCDCLifeExpectency(TestETL):
+    """Tests the CDC Life Expectancy ETL.
+
+    This uses pytest-snapshot.
+    To update individual snapshots: $ poetry run pytest
+            data_pipeline/tests/sources/cdc_life_expectancy/test_etl.py::TestClassNameETL::<testname>
+            --snapshot-update
+    """
+
+    _ETL_CLASS = CDCLifeExpectancy
+
+    _SAMPLE_DATA_PATH = pathlib.Path(__file__).parents[0] / "data"
+    _SAMPLE_DATA_FILE_NAME = "US_A.CSV"
+    _SAMPLE_DATA_ZIP_FILE_NAME = None
+    _EXTRACT_TMP_FOLDER_NAME = "CDCLifeExpectanc"
+    _EXTRACT_CSV_FILE_NAME = "extract.csv"
+    _FIXTURES_SHARED_TRACT_IDS = TestETL._FIXTURES_SHARED_TRACT_IDS + [
+        "55001950201",  # WI
+        "23001010100",  # ME
+    ]
+
+    def setup_method(self, _method, filename=__file__):
+        """Invoke `setup_method` from Parent, but using the current file name.
+
+        This code can be copied identically between all child classes.
+        """
+        super().setup_method(_method=_method, filename=filename)
+
+    def _setup_etl_instance_and_run_extract(
```

Comment:
is there a reason this needs to be re-implemented here, and we can't just use the method from the base class?